de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/Category.java

```diff
@@ -354,84 +323,16 @@ public int getNumberOfPages() {
      * It is used for performance tuning.
      * @return The set of pages that are categorized under this category.
      */
-    protected Set<Integer> __getPages() {
-        Session session = this.wiki.__getHibernateSession();
-        session.beginTransaction();
-        session.lock(hibernateCategory, LockMode.NONE);
-        Set<Integer> tmpSet = new HashSet<Integer>(hibernateCategory.getPages());
-        session.getTransaction().commit();
-        return tmpSet;
+    Set<Integer> __getPages() {
```

Comment:
In this case the reasons is: 
No piece of code from "external" packages used it, merely internal consumers. Moreover the "__" in front of `getPages()` indicates the internal use, I guess. 

If there is a necessity to change it back, I can do that. Otherwise I suggest to leave it like that, as it is "internal" API here.